### DISCUSSION TITLE
How to pragmatically get authentication object in any method

### DISCUSSION THREAD

**Discussion by siddharth223437** [micronaut-projects/micronaut-core#7429] [Q&A] [answered]
How to pragmatically get authentication object in any method ?

In springboot i used

 `securitycontextholder.getcontext().getauthentication() `

**Comment by morki** [comment, marked as answer]
You can use [SecurityService](https://micronaut-projects.github.io/micronaut-security/latest/api/io/micronaut/security/utils/SecurityService.html).
